addons/im_livechat/static/src/legacy/public_livechat.scss

```diff
@@ -4,6 +4,12 @@
 
 $o-mail-thread-window-zindex: $zindex-modal + 1 !default;
 
+@include media-breakpoint-down(md) {
+    .o_frontend_to_backend_nav:not(.d-none) ~ .o_thread_window .o_thread_window_title {
```

Comment:
Could you move this chuck inside the main selector? There is already a media-query there that suits the purpose.
Something like 
```scss
.o_thread_window {
  [...]
  
  @include media-breakpoint-down(md) {
      width: 100%;
      height: 100%!important;
      box-shadow: none;
      
      .o_frontend_to_backend_nav:not(.d-none) ~ & .o_thread_window_title {
           [...]
      }
   }